Makefile

```diff
@@ -64,18 +77,34 @@ generate-manifests:
 	SRIOV_DP_IMAGE=$(SRIOV_DP_IMAGE) \
 	SRIOV_CNI_IMAGE=$(SRIOV_CNI_IMAGE) \
 	KUBEMACPOOL_IMAGE=$(KUBEMACPOOL_IMAGE) \
-		./hack/build-manifests.sh
+		./hack/generate-manifests.sh
+
+generate-api:
```

Comment:
I have named it gen-k8s since it's not generating the API it's generating the DeepCopy stuff.

Also let's track source code si we can call this always before build without much time penalty

gen-k8si: $(apis_sources)
operator-sdk generate k8s
touch $@

Reply:
Done